test/test_simulation_results.jl

```diff
@@ -252,123 +252,150 @@ function test_simulation_results(file_path::String, export_path; in_memory = fal
             end
         end
 
-        realized_variable_uc = read_realized_variables(results_uc)
-        @test length(keys(realized_variable_uc)) == 12
-        for var in values(realized_variable_uc)
-            @test size(var)[1] == 48
-        end
-
-        realized_param_uc = read_realized_parameters(
-            results_uc,
-            names = [:P__max_active_power__RenewableDispatch_max_active_power],
-        )
-        @test length(keys(realized_param_uc)) == 1
-        for param in values(realized_param_uc)
-            @test size(param)[1] == 48
-        end
-
-        realized_duals_ed = read_realized_duals(results_ed)
-        @test length(keys(realized_duals_ed)) == 1
-        for param in values(realized_duals_ed)
-            @test size(param)[1] == 576
-        end
+        # TODO: is read_realized_variables still supported?
```

Comment:
@daniel-thom yes this should be working again. Not sure if the code will be still valid. 


@claytonpbarrows we should discuss a new name for this function since `realized_variables` should come from the emulation results.